File: Top 100 Codes/001-check if a Number Is Positive Or Negative/Method 2- Using Nested if-else Statements/PositiveAndNegativeNumber.c
#include<stdio.h>

int main()
{
    int num ;
    //Take input
    printf("Enter a number: ");
    scanf("%d",&num);

    //Condition to check if num is negative/positive or zero
    if(num >= 0)
    {
        if (num == 0)
            printf("The number is 0");
        else
            printf("The number is positive");
    }
    else
       printf("The number is negative");

    return 0;
}

File: Top 100 Codes/001-check if a Number Is Positive Or Negative/Method 2- Using Nested if-else Statements/Positive_And_Negative_Number.java
import java.util.Scanner;
public class Positive_And_Negative_Number {
    public static void main(String[] args) {
        //using Nested If-else Statement
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a Number: ");
        int num = sc.nextInt();

        //Condition to check if the number is positive or negative
        if(num >=0)
        {
            if(num == 0)
                System.out.println("Zero");
            else 
                System.out.println("The number is positive");

        }
        else
            System.out.println("The number is Negative");

        sc.close();
        
    }
    
}
